sysdig/common/client.go

```diff
@@ -13,6 +14,11 @@ type SysdigCommonClient interface {
 	GetUserById(int) (User, error)
 	DeleteUser(int) error
 	UpdateUser(User) (User, error)
+
+	CreateTeam(Team) (Team, error)
```

Comment:
Teams in monitor are not the same teams that ones in secure so that this should not be in Common.

@tembleking implemented teams support for monitor in: https://github.com/draios/terraform-provider-sysdig/commit/1de6ebf2df9b68bf34b745dcf7db7b814ef3925a